diff --git a/packages/frontend/src/hooks/use-editor-props.tsx b/packages/frontend/src/hooks/use-editor-props.tsx
index 7785447..acb21fe 100644
--- a/packages/frontend/src/hooks/use-editor-props.tsx
+++ b/packages/frontend/src/hooks/use-editor-props.tsx
@@ -358,13 +358,13 @@ export function useEditorProps(
          * Runtime plugin
          */
         createRuntimePlugin({
-          mode: 'browser',
-          // mode: 'server',
-          // serverConfig: {
-          //   domain: 'localhost',
-          //   port: 4000,
-          //   protocol: 'http',
-          // },
+          // mode: 'browser',
+          mode: 'server',
+          serverConfig: {
+            domain: 'localhost',
+            port: 4000,
+            protocol: 'http',
+          },
         }),
 
         /**
